doc: add minutes for meeting 29 Oct 2020 (#946)
nodejs/TSC: meetings/2020-10-29.md

# Node.js Technical Steering Committee (TSC) Meeting 2020-10-29

## Links

* **Recording**: https://youtu.be/tKQRw8fAClo  
* **GitHub Issue**: https://github.com/nodejs/TSC/issues/942

## Present

* Beth Griggs @BethGriggs (TSC)
* Colin Ihrig @cjihrig (TSC)
* Gabriel Schulhof @gabrielschulhof (TSC)
* Gireesh Punathil @gireeshpunathil (TSC)
* Michael Dawson @mhdawson (TSC)
* Myles Borins @MylesBorins (TSC)
* Michaël Zasso @targos (TSC)
* Tobias Nießen @tniessen (TSC)
* Rich Trott @Trott (TSC)

## Agenda

### Announcements

* Gabriel - new webidl-NAPi project to translate webidl to NAPI code
* Michael - 14.x went LTS this week !

### CPC and Board Meeting Updates
 
*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.

* Board meeting this week. Nothing new that I have to bring up with them.

### nodejs/node

* [34895](https://github.com/nodejs/node/pull/34895), blocked on on TSC approvals.
 * new top module -> diagnostics_channel

>  * @Quard owns namespace in npm, so no chance of collision

Reviewer: The author of <https://github.com/nodejs/node/pull/34895> is @Qard, [Quard](https://github.com/Quard) is someone unrelated to the @nodejs project.
```suggestion
 * @Qard owns namespace in npm, so no chance of collision
```

 * Needs 2 TSC approvals
   * No objections/concerns raised in the meeting.

* Backporting `util/types`, `path/posix` and `path/win32` to v15 proposal [#35788](https://github.com/nodejs/node/issues/35788)
  * Myles, concern not with this change, but policy around new named modules, /xxx promises
    is getting in the way. We should find some sort of pattern, or be ok with more ecosystem 
    Breakage.
  * Rich past discussion, but no agreement on what the namespace should be. 

>   * Myles, have started using nodejs: loader internally. Can be used with ES Modules, 

Reviewer: The scheme is `node:`, not `nodejs:` (see <https://github.com/nodejs/node/pull/35387>)
```suggestion
  * Myles, have started using node: loader internally. Can be used with ES Modules, 
```

    natural next step would be to port that over to CommonJS.
  * Michael, since we have a name no reason to use anything anything.

* Rename default branch from "master" to "main" or something similar [#33864](https://github.com/nodejs/node/issues/33864)
  * No update this week.
 
### nodejs/TSC

* More visibility for TSC on performance issues [#940](https://github.com/nodejs/TSC/issues/940)
  * Some discussion around if this makes any sense
  * Tracking issue that occasionally put on the agenda?

### nodejs/admin

* Audit Google account access [#389](https://github.com/nodejs/admin/issues/389)
  * no update on that.

### nodejs/security-wg

* Potential stagnation of open issues on h1 bounty program [#654](https://github.com/nodejs/security-wg/issues/654)
  * Discussion, general support from those in the meeting that TSC members
    support shutting down the program. 
  * Myles can check if they have a replacement at GitHub for places people
    can report vulnerabilities.

>   * Rich will reach out to those on the team to discuss winding it down.

Reviewer 2: ```suggestion
  * Rich will reach out to some involved parties on the team to discuss winding it down.
```
